settings.py

```diff
@@ -1,8 +1,18 @@
 import os
+from distutils import util
 
 APP_PORT = os.environ.get('APP_PORT') or 80
 
 DB_HOST = os.environ.get('DB_HOST') or "localhost"
 DB_NAME = os.environ.get('DB_NAME')
 USER = os.environ.get('DB_USER')
 PASSWD = os.environ.get('DB_PASSWORD')
+
+# util.strtobool
+# True values are y, yes, t, true, on and 1;
+# False values are n, no, f, false, off and 0.
+# Raises ValueError if val is anything else.
+RESTRICT_BY_IP_ENABLED = util.strtobool(
+    os.environ.get('RESTRICT_BY_IP_ENABLED') or 'yes'
+)
+ALLOWED_IP_ADDRESSES = os.environ.get('ALLOWED_IP_ADDRESSES') or ''
```

Comment:
It's a bit inconsistent - `RESTRICT_BY_IP_ENABLED` is in python type (boolean), but `ALLOWED_IP_ADDRESSES` is not yet converted into python array.

They could be of the same level, i.e. both values raw values from ENV (two strings) or both values converted (boolean and array of strings)

Reply:
converted ALLOWED_IP_ADDRESSES to array